[2025-04-08T11:18:12.373+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-08T11:18:12.413+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bash_operator.print_hello manual__2025-04-08T11:18:08.874906+00:00 [queued]>
[2025-04-08T11:18:12.435+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bash_operator.print_hello manual__2025-04-08T11:18:08.874906+00:00 [queued]>
[2025-04-08T11:18:12.437+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-08T11:18:12.469+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): print_hello> on 2025-04-08 11:18:08.874906+00:00
[2025-04-08T11:18:12.480+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=415) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-08T11:18:12.483+0000] {standard_task_runner.py:72} INFO - Started process 416 to run task
[2025-04-08T11:18:12.484+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'bash_operator', 'print_hello', 'manual__2025-04-08T11:18:08.874906+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/__pycache__/bash_operator.py', '--cfg-path', '/tmp/tmpli5nff5e']
[2025-04-08T11:18:12.486+0000] {standard_task_runner.py:105} INFO - Job 7: Subtask print_hello
[2025-04-08T11:18:12.630+0000] {task_command.py:467} INFO - Running <TaskInstance: bash_operator.print_hello manual__2025-04-08T11:18:08.874906+00:00 [running]> on host e8bb6e8ec07a
[2025-04-08T11:18:14.863+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bash_operator' AIRFLOW_CTX_TASK_ID='print_hello' AIRFLOW_CTX_EXECUTION_DATE='2025-04-08T11:18:08.874906+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-08T11:18:08.874906+00:00'
[2025-04-08T11:18:14.865+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-08T11:18:14.981+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-08T11:18:14.984+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', "echo '¡Hola desde Docker!'"]
[2025-04-08T11:18:15.003+0000] {subprocess.py:99} INFO - Output:
[2025-04-08T11:18:15.019+0000] {subprocess.py:106} INFO - ¡Hola desde Docker!
[2025-04-08T11:18:15.020+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-08T11:18:15.063+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-08T11:18:15.064+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=bash_operator, task_id=print_hello, run_id=manual__2025-04-08T11:18:08.874906+00:00, execution_date=20250408T111808, start_date=20250408T111812, end_date=20250408T111815
[2025-04-08T11:18:15.122+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-08T11:18:15.154+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-08T11:18:15.160+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
